### ISSUE TITLE
Incompatible with latest MonoGame 3.7

### ISSUE THREAD

**Issue by BrettStory** [MarcStan/MonoGame.Framework.WpfInterop#11]
MonoGame recently updated to 3.7, which now uses SharpDX 4.01. An error occurs when using the latest WpfInterop nuget package with the latest MonoGame package. I realize that as of right now, you've only guaranteed WpfInterop working with 3.6, so you can mark this as an enhancement. I've cloned the source personally and tested after upgrading WpfInterop to use the latest MonoGame nuget package and I don't notice any issues. Up to you whether or not to do anything about this, but I just thought I'd let you know.

Thanks.

**Comment by MarcStan** [comment]
Thanks for letting me know Brett.

I actually started upgrading to support 3.7 yesterday evening as it lets me fix two longstanding issues ([MSAA](https://gitlab.com/MarcStan/MonoGame.Framework.WpfInterop/issues/13) and [DPI scaling](https://gitlab.com/MarcStan/MonoGame.Framework.WpfInterop/issues/17)).

However I ran into a small edge case where MonoGame crashes with a null reference exception on disposing one instance (only happes if you have one MSAA and one non-MSAA scene and the MSAA instance gets disposed). I'll see if I can get it fixed today and release a new version.
